test/unit/mods.example.ts

```diff
@@ -264,4 +264,92 @@ export const ArchiveOnly = new Map<string, ExampleMod>(
   }), // object
 );
 
-export const AllModTypes = [CetMod, ArchiveOnly];
+export const jsonMod = new Map<string, ExampleMod>(
```

Comment:
Super minor - I was wondering why the linter didn't pick up on this but of course it doesn't, have to see if there's a switch we could use.

Since this is a 'real' const, CamelCase it? :)

Reply:
No problem :)